pkg/virt-handler/vm_test.go

```diff
@@ -1941,8 +1944,9 @@ var _ = Describe("VirtualMachineInstance", func() {
 				{
 					Name: interfaceName,
 					Mac:  mac,
-					Ip:   ip,
-					IPs:  ips,
+					// Qemu guest agent reports CIDR
```

Comment:
This is specified in the variable names, unless there is a hidden added info here, I would suggest to drop the comment.